src/core/ext/transport/binder/wire_format/wire_writer.cc

```diff
@@ -77,4 +77,13 @@ absl::Status WireWriterImpl::RpcCall(const Transaction& tx) {
   // is an undefined behavior.
   return binder_->Transact(BinderTransportTxCode(tx.GetTxCode()));
 }
+
+absl::Status WireWriterImpl::Ack(size_t num_bytes) {
```

Comment:
We reports total number of bytes received in this RPC call?

Again, we better document this in wire format doc..

Reply:
Yeah I didn't find any document about this behavior. This is just what the Java implementation did.

Reply:
I've added this to the gRFC PR.